src/riverroute/RtmMod.F90

```diff
@@ -130,56 +131,17 @@ subroutine Rtmini(flood_active)
-    real(r8) :: dy                            ! lat dist. betn grid cells (m)
-    real(r8) :: lrtmarea                      ! tmp local sum of area
-    real(r8),allocatable :: tempr(:,:)        ! temporary buffer
-    integer ,allocatable :: itempr(:,:)       ! temporary buffer
-    integer ,allocatable :: idxocn(:)         ! downstream ocean outlet cell
-    integer ,allocatable :: nupstrm(:)        ! number of upstream cells including own cell
-    integer ,allocatable :: pocn(:)           ! pe number assigned to basin
-    integer ,allocatable :: nop(:)            ! number of gridcells on a pe
-    integer ,allocatable :: nba(:)            ! number of basins on each pe
-    integer ,allocatable :: nrs(:)            ! begr on each pe
-    integer ,allocatable :: basin(:)          ! basin to mosart mapping
-    integer  :: nmos,nmos_chk                 ! number of mosart points
-    integer  :: nout,nout_chk                 ! number of basin with outlets
-    integer  :: nbas,nbas_chk                 ! number of basin/ocean points
-    integer  :: nrof,nrof_chk                 ! num of active mosart points
-    integer  :: baspe                         ! pe with min number of mosart cells
-    integer  :: maxrtm                        ! max num of rtms per pe for decomp
-    integer  :: minbas,maxbas                 ! used for decomp search
-    integer  :: nl,nloops                     ! used for decomp search
     integer  :: ier                           ! error code
     integer  :: mon                           ! month (1, ..., 12)
     integer  :: day                           ! day of month (1, ..., 31)
-    integer  :: numr                          ! tot num of roff pts on all pes
     real(r8) :: dtover,dtovermax              ! ts calc temporaries
-    type(file_desc_t) :: ncid                 ! netcdf file id
-    integer  :: dimid                         ! netcdf dimension identifier
     integer  :: nroflnd                       ! local number of land runoff 
     integer  :: nrofocn                       ! local number of ocn runoff
-    integer  :: pid,np,npmin,npmax,npint      ! log loop control
     integer  :: na,nb,ns                      ! mct sizes
     integer  :: ni,no,go                      ! tmps
-    integer ,pointer  :: rgdc2glo(:)          ! temporary for initialization
-    integer ,pointer  :: rglo2gdc(:)          ! temporary for initialization
-    integer ,pointer  :: gmask(:)             ! global mask
-    logical           :: found                ! flag
-    character(len=256):: fnamer               ! name of netcdf restart file 
-    character(len=256):: pnamer               ! full pathname of netcdf restart file
-    character(len=256):: locfn                ! local file name
-    character(len=16384) :: rList             ! list of fields for SM multiply
     integer           :: unitn                ! unit for namelist file
 #ifdef NDEBUG
```

Comment:
It looks like the NDEBUG setting can be removed from this one. Possibly other things can be removed?